backend/coins/eth/account.go

```diff
@@ -961,3 +966,16 @@ func (account *Account) EthSignWalletConnectTx(
 	}
 	return "0x" + hex.EncodeToString(txHash[:]), "0x" + hex.EncodeToString(rawTx), nil
 }
+
+// Returns the account's single Ethereum address.
+func (account *Account) Address() *Address {
+	if !account.isInitialized() {
+		return &Address{}
```

Comment:
i know the only current use works out with this, but it's more robust and standard to return an error. potentially future use cases could fail silently or in hard to debug ways by getting a "valid" but wrong address.

Reply:
yeah, that was mostly laziness on my part since I didn't want to rewrite
`if ethcommon.HexToAddress(address).Hex() == ethAccount.Address().Hex() && !eth.IsERC20(ethAccount)` to get the address first and check for errors :D

Reply:
It's the same in: 
```
func (account *Account) GetUnusedReceiveAddresses() []accounts.AddressList {
	if !account.isInitialized() {
		return nil
	}
	return []accounts.AddressList{{
		Addresses: []accounts.Address{account.address},
	}}
}
``` :P

Reply:
I thought that was the reason ;)

GetUnusedReceiveAddresses should be converted to you are right :sweat_smile: